services-gateway-client-transport/src/main/java/io/scalecube/services/gateway/transport/websocket/WebsocketGatewayClient.java

```diff
@@ -24,8 +26,8 @@
 
   private static final AtomicReferenceFieldUpdater<WebsocketGatewayClient, Mono>
       websocketMonoUpdater =
-          AtomicReferenceFieldUpdater.newUpdater(
-              WebsocketGatewayClient.class, Mono.class, "websocketMono");
+      AtomicReferenceFieldUpdater.newUpdater(
```

Comment:
Looks like something changed in formatting. Could you please check the formatting settings?